package.json

```diff
@@ -18,10 +18,11 @@
   "typings": "dist/types",
   "exports": {
     ".": {
+      "types": "./dist/types/index.d.ts",
       "import": "./dist/kongponents.es.js",
-      "require": "./dist/kongponents.umd.js",
-      "types": "./dist/types/index.d.ts"
+      "require": "./dist/kongponents.umd.js"
     },
+    "./style.css": "./dist/kongponents.css",
```

Comment:
It seems that we don't have a `kongponents.css` file in our `dist` directory now?

Reply:
Updated